doc/develop/gui_tree.svg

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" fill-opacity="1" color-rendering="auto" color-interpolation="auto" text-rendering="auto" stroke="black" stroke-linecap="square" width="1354" stroke-miterlimit="10" shape-rendering="auto" stroke-opacity="1" fill="black" stroke-dasharray="none" font-weight="normal" stroke-width="1" height="549" font-family="'Dialog'" font-style="normal" stroke-linejoin="miter" font-size="12px" stroke-dashoffset="0" image-rendering="auto">
```

Comment:
I think for most of the arrows, there should be "black diamond" (aka composition). It's also useful to indicate the amount (ie, 1 or *). For example, the StreamBarController has "*" StreamControllers. See
https://en.wikipedia.org/wiki/Class_diagram#Instance-level_relationships